lib/auth/googleauth.js

```diff
@@ -37,9 +37,9 @@ function GoogleAuth() {
 }
 
 // Executes the given callback if it is not null.
-function callback(c, err, res) {
+function callback(c, err, arg1, arg2) {
   if (c) {
-    c(err, res);
+    c(err, arg1, arg2);
```

Comment:
No, a bind will still work even if apply is called on the result. Think of a this-only bind like this:

```js
function bind(fn, self) {
  return function() {
    return fn.call(self, arguments);
  }
}
```

Example of this working:


```js
var fn = (function () { return this.a }).bind({a: "hello"});
console.log(fn.call(null, []));
```

(outputs hello)